cmd/gopcomm/community_yap.gox

```diff
@@ -1,52 +1,364 @@
 import (
 	"context"
+	"time"
+	"os"
+	"io"
+	"net/http"
+	"fmt"
 
 	"github.com/goplus/community/internal/core"
+	"github.com/goplus/community/markdown"
+	"github.com/goplus/community/translation"
+	"github.com/casdoor/casdoor-go-sdk/casdoorsdk"
+	"go.uber.org/zap"
+	language "golang.org/x/text/language"
+	_ "github.com/joho/godotenv/autoload"
 )
 
 var (
 	community *core.Community
+	trans *translation.Engine
 )
 
 todo := context.TODO()
+endpoint := os.Getenv("GOP_COMMUNITY_ENDPOINT")
+domain := os.Getenv("GOP_COMMUNITY_DOMAIN")
+logger, _ := zap.NewProduction()
```

Comment:
handle err scenario

Reply:
for log library, recommend to use https://github.com/qiniu/x/tree/main/xlog

Reply:
I will update to [xlog](https://github.com/qiniu/x/tree/main/xlog).